src/com/tigersale/userInterface/inventoryManager/RemoveProductView.java

```diff
@@ -0,0 +1,29 @@
+package com.tigersale.userInterface.inventoryManager;
+
+import com.tigersale.database.ProductTable;
+import com.tigersale.model.InventoryManager;
+import com.tigersale.userInterface.AbstractView;
+
+import java.util.Scanner;
+
+/**
+ * @author James Haller
+ */
+public class RemoveProductView extends AbstractView {
+
+    public RemoveProductView(Scanner scanner) {
```

Comment:
I would guess this view would disappear as well.  What you could put in is a view for the manager to change anything about the product.  Increase/decrease stock, change descriptions, categories, etc..